samples/host_exerciser/host_exerciser.h

```diff
@@ -307,6 +321,11 @@ class host_exerciser : public test_afu {
     // Delay
     app_.add_option("-d,--delay", he_delay_, "Enables random delay insertion between requests")->default_val(false);
 
+    // how many requests to interleave when TestMode is Throughput
+    app_.add_option("--interleave", he_interleave_, "number of interleave in Throughput mode \
+           {rdwrrdwr, rdrdwrwr, rdrdrdrdwrwrwrwr}")
```

Comment:
Doesn't seem very user friendly.
Arguments like this seem too long and makes it easier for a user to make a mistake when typing in the argument value.
Can we maybe have interleave numbers (1,2,3) and explain in the help string and documentation what those mean?
If you do go with 1,2,3 you'll still need a transform to turn them into 0,1,2.

Reply:
updated interleave  input in readable format 
      value 0:Rd,Wr,Rd,Wr
               1:Rd,Rd,Wr,Wr
               2:Rd,Rd,Rd,Rd,Wr,Wr,Wr,Wr

Reply:
No, this is not acceptable. Please use short input argument value choices.
A user shouldn't be expected to actually type something like "rdwrrdwr" or "rdrdrdrdwrwrwrwr".
For things like this, numeric codes are preferable Then, have the help string and documentation interpret the codes and what they mean.

Reply:
app_.add_option("--interleave", he_interleave_, "number of interleave in Throughput mode {0, 1, 2}")
       ->default_val(0);

updated to short input arguments {0,1,2}